schema/project-level/codelists/+documentType.csv

```diff
@@ -0,0 +1,6 @@
+Code,Title,Description
```

Comment:
There is both `+documentType.csv` and `documentType.csv` in the same folder. Shouldn't only one be retained?

Reply:
See https://github.com/open-contracting/infrastructure/issues/25